docs/src/bidding_formats.md

```diff
@@ -17,24 +17,239 @@ Note that, in case the heuristic bid alternative is chosen, bid files will be ge
 
-Note that in the examples above we assume that the quantity file `"q"` and the price file `"p"` follow the correct structure specified above for each bid type. It is also worth highlighting that all three types of bid also have their time series varying per [period, scenario, and segment](key_features.md#glossary), in addition to potentially varying per subperiod.
+### Independent bids:
 
-## Complex profile bids
+`independent_bids` have both quantities and prices varying per [subperiod](key_features.md#glossary), as decisions are indeed individualized per subperiod. Independent bids can be parameterized using the following syntax: `link_time_series_to_file(db,"BiddingGroup"; quantity_offer = "q", price_offer = "p")`, where `"q"` and `"p"` are the names of the CSV files containing the time series data for the quantity and price offers, respectively. The expected structure of these files is shown in the following tables:
+
+
+#### Price offer
+
+
+| period | scenario | subperiod | bid_segment | bg_1 - bus_1 | bg_1 - bus_2 | bg_2 - bus_1 | bg_2 - bus_2 |
+|:------:|:--------:|:---------:|:-----------:|:------------:|:------------:|:------------:|:------------:|
+|   1    |    1     |     1     |      1      |    100.0     |     80.0     |     90.0     |     70.0     |
+|   1    |    1     |     2     |      1      |    100.0     |     80.0     |     90.0     |     70.0     |
+|   1    |    2     |     1     |      1      |    100.0     |     80.0     |     90.0     |     70.0     |
+|   1    |    2     |     2     |      1      |    100.0     |     80.0     |     90.0     |     70.0     |
+
+
+
+
+#### Quantity offer
+
+
+| period | scenario | subperiod | bid_segment | bg_1 - bus_1 | bg_1 - bus_2 | bg_2 - bus_1 | bg_2 - bus_2 |
+|:------:|:--------:|:---------:|:-----------:|:------------:|:------------:|:------------:|:------------:|
+|   1    |    1     |     1     |      1      |     5.0      |     1.0      |     5.0      |     2.0      |
+|   1    |    1     |     2     |      1      |     5.0      |     1.0      |     5.0      |     2.0      |
+|   1    |    2     |     1     |      1      |     4.0      |     1.5      |     4.0      |     3.0      |
+|   1    |    2     |     2     |      1      |     4.0      |     1.5      |     4.0      |     3.0      |
+
+
+
+### Profile bids:
+
+`profile bids` have quantities varying per subperiod but prices not: the decision on whether or not to activate the profile bid is made only once in the period (and therefore it is sufficient to represent the associated cost with a single price parameter), but potentially affects all subperiods. Profile bids can be parameterized using the following syntax: `link_time_series_to_file(db,"BiddingGroup"; quantity_offer_profile = "q", price_offer_profile = "p")`, where `"q"` and `"p"` are the names of the CSV files containing the time series data for the quantity and price offers, respectively.
```

Comment:
Mesma coisa do `independent_bids`

Reply:
but potentially affects all subperiods -> acho que podia tirar
time series data for the quantity and price offers, respectively. -> time series data for the quantity and price profile offers, respectively.